### DISCUSSION TITLE
Can't build fluidsynth on macos ventura 13.5.2

### DISCUSSION THREAD

**Discussion by bclark288** [FluidSynth/fluidsynth#1303] [Q&A] [answered]
I'm trying to build fluidsynth on macos ventura 13.5.2.  The cmake command fails with the following error

CMake Error at src/CMakeLists.txt:290 (add_library):
  Target "libfluidsynth" links to target "MPG123::libmpg123" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?

libmgp123 is installed at /usr/local/lib/libmpg123.0.dylib
Any ideas on how to proceed?

**Comment by derselbst** [comment, marked as answer]
1. Which version of fluidsynth are you trying to build?
2. Delete and re-create the build directory to get a clean build.
3. Then, post the _full_ output of CMake's console log, as well as the `CMakeError.log` and `CMakeOutput.log` files.
4. Fluidsynth itself does not depend on mpg123, but libsndfile may. To confirm that the error is coming from there, run cmake again with the `-Denable-libsndfile=0` flag, and the build should succeed.

**Reply by bclark288** [reply]
That did the trick!  Thanks for the quick (and accurate) response.
